packages/wrangler/package.json

```diff
@@ -95,6 +95,7 @@
   ],
   "scripts": {
     "clean": "rm -rf wrangler-dist",
+    "check:type": "tsc -p ../../tsconfig.json",
```

Comment:
I wonder if we should just move that tsconfig.json here?

Reply:
Could do. Thought it might be re-used with `create-workers-app` though?